features/ios/ios_native_errors.feature

```diff
@@ -1,65 +1,114 @@
-        And the event "app.releaseStage" equals "production"
-        And the event "app.type" equals "iOS"
-        And the event "app.version" equals "1.2.3"
-        And the event "app.bundleVersion" equals "1.2.3"
-        And the event "app.isLaunching" is true
-        And the event "app.inForeground" is true
-        And the event "app.durationInForeground" is greater than 0
+        #App metadata
         And the event "app.duration" is greater than 0
-        And the event "app.binaryArch" is not null
+        And the event "app.durationInForeground" is not null
+        And the event "app.inForeground" is not null
+        And the event "app.isLaunching" is not null
+        And the event "app.releaseStage" is not null
+        And the event "app.type" is not null
+        And the event "app.version" is not null
+        And the event "metaData.app.companyName" equals "bugsnag"
+        And the event "metaData.app.name" matches ".azerunner"
+        And the event "metaData.app.buildno" is not null
 
-        # Device data
-        And the event "device.jailbroken" is false
-        And the event "device.id" is not null
-        And the event "device.locale" is not null
-        And the event "device.manufacturer" is not null
-        And the event "device.model" is not null
-        And the event "device.osName" equals "iOS"
-        And the event "device.osVersion" is not null
-        And the event "device.runtimeVersions" is not null
-        And the event "device.totalMemory" is not null
-        And the event "device.freeDisk" is not null
-        And the event "device.freeMemory" is not null
-        And the event "device.time" is not null
+        # Exception details
+        And the error payload field "events" is an array with 1 elements
+        And the exception "errorClass" equals "St13runtime_error"
+        And the exception "message" equals "CocoaCppException"
+        And the event "unhandled" is true
+        And the event "severity" equals "error"
 
+        # Metadata
+        And the event "metaData.init" is null
+        And the event "metaData.custom.letter" equals "QX"
+        And the event "metaData.custom.better" equals 400
+        And the event "metaData.test.test1" equals "test1"
+        And the event "metaData.test.test2" is null
 
-    Scenario: Last Run Info
-        When I run the "iOS Native Error" mobile scenario
+    Scenario: iOS signal Smoke Test
+        When I run the game in the "IosSignal" state
         And I wait for 2 seconds
-        And I relaunch the Unity mobile app
-        When I run the "Check Last Run Info" mobile scenario
-        Then I wait to receive 2 errors
-        And I discard the oldest error
-        And the exception "message" equals "Last Run Info Correct"
\ No newline at end of file
+        And I clear any error dialogue
+        And On Mobile I relaunch the app
+        And I run the game in the "StartSDKDefault" state
+        And I wait for 3 seconds
```

Comment:
No need to wait a few seconds before the next step, as it waits anyway.

Reply:
Done 👍